src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,584 @@
+        )
+        .then(function(resp) {
+            console.log(resp);
+            return resp.text();
+        })
+        .then(function(respText) {
+            var schema = JSON.parse(respText);
+            complete(schema);
+        });
+    // */
+}
+
+/** Path to images directory/CDN. Once is published to NPM, will change to unpkg CDN URL. */
+const imagesPathSVG = "https://raw.githubusercontent.com/WU-BIMAC/4DNMicroscopyMetadataToolReact/master/public/assets/svg/";
+const imagesPathPNG = "https://raw.githubusercontent.com/WU-BIMAC/4DNMicroscopyMetadataToolReact/master/public/assets/png/";
+
+let MicroscopyMetadataTool = null;
+
+export class MicroMetaTabView extends React.PureComponent {
+
+    static getTabObject(props, width) {
+        return {
+            'tab' : <span><i className="icon icon-microscope fas icon-fw"/> MicroMeta</span>,
+            'key' : 'micrometa',
+            //'disabled' : !Array.isArray(context.experiments),
+            'content' : (
+                <div className="overflow-hidden">
+                    <h3 className="tab-section-title">
+                        <span>4DN MicroMeta</span>
+                    </h3>
+                    <hr className="tab-section-title-horiz-divider"/>
+                    <MicroMetaTabView {...props} width={width} />
+                </div>
+            )
+        };
+    }
+
+    constructor(props){
+        super(props);
+        this.fullscreenButton = this.fullscreenButton.bind(this);
+        this.saveButton = this.saveButton.bind(this);
+        this.cloneButton = this.cloneButton.bind(this);
+        this.havePermissionToEdit = this.havePermissionToEdit.bind(this);
+        this.handleFullscreenToggle = this.handleFullscreenToggle.bind(this);
+        this.handleSave = _.throttle(this.handleSave.bind(this), 3000);
+        this.handleClone = _.throttle(this.handleClone.bind(this), 3000);
+        this.handleModalCancel = _.throttle(this.handleModalCancel.bind(this), 3000);
+        this.handleStatusChange = this.handleStatusChange.bind(this);
+        this.onSaveMicroscope = this.onSaveMicroscope.bind(this);
+        this.getMicroscopyMetadataToolComponent = this.getMicroscopyMetadataToolComponent.bind(this);
+        this.updateContainerOffsets = _.debounce(this.updateContainerOffsets.bind(this), 500);
+
+        this.state = {
+            'mounted'               : false,
+            'saveLoading'           : false,
+            'cloneLoading'          : false,
+            'releaseLoading'        : false,
+            'addFileLoading'        : false,
+            'modal'                 : null,
+            'containerOffsetLeft'   : 0,
+            'containerOffsetTop'    : 0
+        };
+
+        this.microMetaToolRef = React.createRef();
+        this.containerElemRef = React.createRef();
+    }
+
+    componentDidUpdate(pastProps, pastState){
+        const { isFullscreen, windowWidth, windowHeight } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['windowHeight' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)